tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -1256,7 +1256,7 @@ def test_functional_python_model_callable_object(tmp_path):
     assert loaded_model.predict(["a", "b"]) == ["a", "b"]
 
 
-def list_to_list(x: List[str]) -> List[str]:
+def list_to_list(x: List[str]) -> List[str]:  # UP006
     return x
```

Comment:
This function is used to test signature enforcement from type hints which supports both `list` and `List`.